src/inference/Inference.jl

```diff
@@ -463,7 +463,6 @@ include("gibbs_conditional.jl")
 include("gibbs.jl")
 include("../contrib/inference/sghmc.jl")
 include("emcee.jl")
-include("../contrib/inference/dynamichmc.jl")
```

Comment:
I think we needed this. 
 Otherwise, It cannot find the functions defined in `"../contrib/inference/dynamichmc.jl"`